### ISSUE TITLE
PktGen only sends packet with ping4, not in bursts

### ISSUE THREAD

**Issue by Ramla-I** [pktgen/Pktgen-DPDK#40]
I'm trying to use pktgen to send 64 byte raw ethernet packets from a single core. 
I can start pktgen, and when I type `start 0`, I can see the stats for opackets and obytes are updated in the `page stats` page.
But I don't receive any packets at the other end.
But when I send a packet using the `ping4` command, I can see that a package has been received at the receiving end (using tcpdump).

I'm not sure what I'm doing wrong, and I would appreciate any input.
I've attached everything that is printed out when I start pkt gen.

I'm running pktgen on a 4 core(8 smt) intel cpu with ubuntu 18.04.
Both dpdk(20.11-rc4) and pktgen (20.11.3) are the latest versions.

[pktgen.txt](https://github.com/pktgen/Pktgen-DPDK/files/5578856/pktgen.txt)

**Comment by pktgen** [comment]
I looked at the destination MAC and it is all zeros. The reason ping works is it sends a broadcast MAC address.

**Comment by Ramla-I** [comment]
Thanks. I've also tried updating the destination ip address and mac address to the receiving machine's, and I've tried sending a raw Ethernet packet from a pcap file. Both times I wasn't receiving anything.
I essentially just want to send 64B raw ethernet frames without any of the network stack processing.

**Comment by pktgen** [comment]
Did you mean to enable '-j' or jumbo frames? Have you tried without using -j option.

In the pktgen.txt file I do not see any frames being transmitted or were these captures of the screen before you did a 'start 0' command?

Normally if pktgen is showing it is sending traffic meaning the out packets are increasing it is sending traffic.

Did you use tcpdump or wireshark to capture of show the received packets in the other host. Also look at netstat statistics and see if the stack is dropping the packets.

If wireshark shows the packets, as it put the packet in promiscuous mode, then it should be working unless the frame is not correct.

**Comment by Ramla-I** [comment]
Thanks for your help.
I removed the -j option and it doesn't make a difference. I've also checked with netstat and no packets are being dropped.
The main page doesn't show any updates once I start sending packets with 'start 0', but it does with a 'ping4' command.
The stats page is updated when I type 'start 0'.
I don't understand why the hardware counters would be updated if the packet is not sent.

I am using tcpdump to check if I'm receiving any packets.
[page_stats.txt](https://github.com/pktgen/Pktgen-DPDK/files/5579896/page_stats.txt)
